docs/articles/expensify-classic/getting-started/Create-a-workspace-for-yourself.md

```diff
@@ -0,0 +1,223 @@
+---
+title: Create a workspace for yourself
+description: Get started with Expensify by creating an individual workspace for yourself
+---
+<div id="expensify-classic" markdown="1">
```

Comment:
Hmm what does this do? I don't see it anywhere else in our codebase?

Reply:
It adds this selector. You can toggle between olddot and newdot on the same page.

<img width="396" alt="image" src="https://github.com/Expensify/App/assets/29673073/753736e4-c3ba-4fc9-8f8d-eaba6914ba14">

Reply:
it can be abstracted to a jekyll command but i didn't do it yet to ship faster

Reply:
oooh niftay!!! thanks for showing me that!!